third_party/2and3/click/utils.pyi

```diff
@@ -74,7 +73,7 @@ class KeepOpenFile:
 
 
 def echo(
-    message: Optional[str] = None,
+    message: Optional[Text] = None,
```

Comment:
This should actually be `Union[bytes, Text]`, because the function accepts bytes in Python 3.